ui/src/test/resources/user_response.json

```diff
@@ -0,0 +1,26 @@
+{
+  "token": "usr-f9154016-94e8-4686-a840-075688ac07b5",
+  "status": "PRE_ACTIVATED",
+  "createdOn": "2017-10-30T22:15:45",
+  "clientUserId": "CSK7b8Ffch",
```

Comment:
can we use some other value? like 123456 ?

Reply:
@fmattos-hw Missed. Done.